src/features/MovieList/MovieTile/styled.js

```diff
@@ -66,27 +68,31 @@ export const Year = styled.p`
     }
 `;
 
-export const TagsContainer = styled.div`
+export const TagsContainer = styled.ul`
     margin-top: 8px;
     display: flex;
     flex-wrap: wrap;
     gap: 8px;
+    padding: 0;
+    list-style: none;
+    margin-bottom: 0;
 `;
 
-export const Tag = styled.p`
+export const Tag = styled.li`
     margin: 0px;
```

Comment:
You dont need units when setting margin to 0

Reply:
fixed